src/services/Sso.php

```diff
@@ -148,79 +167,99 @@ function afterDeleteUser($userName){
         $RequestUrl   =  $forumUrl."/register";
         $result       = Websitetoolboxforum::getInstance()->sso->sendApiRequest('POST',$RequestUrl,$postData,'json');
     }
-    function afterLogOut(){  
+    function afterLogOut(){
       if(isset($_COOKIE['forumLogoutToken'])){
+        $cookieForumLogoutToken = $_COOKIE['forumLogoutToken'];
         $forumUrl     = Craft::$app->getProjectConfig()->get('plugins.websitetoolboxforum.settings.forumUrl',false);
-        echo '<img src='.$forumUrl.'/register/logout?authtoken='.$_COOKIE['forumLogoutToken'].' border="0" width="0" height="0" alt="" id="imageTag">';   
-        //Websitetoolboxforum::getInstance()->sso->resetCookieOnLogout();  
+        echo '<img src='.$forumUrl.'/register/logout?authtoken='.$cookieForumLogoutToken.' border="0" width="0" height="0" alt="" id="imageTag">'; 
       }
     }
     function resetCookieOnLogout(){
+      setcookie('forumLogInToken', 0, time() - 3600, "/");
       setcookie('forumLogoutToken', 0, time() - 3600, "/");
       setcookie('forumLoginUserid', '', time() - 3600, "/");
       setcookie('loginRemember', '', time() - 3600, "/");
-   }
-   function renderJsScriptEmbedded($forumUrl,$userStatus){ 
-        if(isset($_COOKIE['forumLogoutToken'])){
-            $cookieForumLogoutToken = $_COOKIE['forumLogoutToken'];
-        }else{
-            $cookieForumLogoutToken = 0;
-        }
+      setcookie('logInForum', '', time() - 3600, "/");
+   } 
+      
+   function renderJsScriptEmbedded($forumUrl,$userStatus)
+   {
+        if(isset($_COOKIE['forumLogInToken']) && $_COOKIE['forumLogInToken'] != ''){
+            $cookieForumLoginToken = $_COOKIE['forumLogInToken'];            
+            setcookie("forumLogInToken", '', time() - 3600,"/");            
+            echo '<img src='.$forumUrl.'/register/dologin?authtoken='.$cookieForumLoginToken.' style="width:0px !important;height:0px !important;" border="0" alt="" id="imageTag">';
```

Comment:
style="width:0px !important;height:0px !important;" not required.

Reply:
Removed